surveys/static/css/custom.css

```diff
@@ -144,25 +125,38 @@ span {
   font-size: .9em;
   display: block;
   text-align: left;
-  color: var(--darker-green-color);
+  color: black;
   background: none;
   font-weight: 600;
   border-radius: 0px;
 }
 
 .sidebar ul li a:hover {
-    color: var(--dark-green-color);
-    background: var(--darker-green-color);
+    padding: 5px 25px;
+}
+
+.sidebar-blurb a {
+  background: var(--off-white-color);
+  border-radius: 5px;
+  padding: 3px 7px;
+}
```

Comment:
I like this styling for the login/logout buttons, but when the links (to DataMade and UCD) have the same styling it makes it a little bit hard to figure out what's an action button and what's a link:

![Screenshot from 2019-05-21 18-25-40](https://user-images.githubusercontent.com/14170650/58182535-c5087c00-7c7b-11e9-978b-2196bd5fa03b.png)

It'd be nice to mute the styles of the links a bit -- they're great to have, but they're not as semantically important in this context.